### PR TITLE
Create a new file for testing the rolling shutdown and recovery of the OCS worker nodes when using MS

### PR DESCRIPTION
**Pull request by yitzhak12**

### CODE REVIEW

File: tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        def finalizer():
+            ocp_nodes = get_node_objs()
+            for n in ocp_nodes:
+                recover_node_to_ready_state(n)
+
+            config.switch_ctx(self.orig_index)
+
+        request.addfinalizer(finalizer)
+
+    @managed_service_required
```
Comment (prsurve) [inline tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py:76]: why can we add this marker at top ?

Comment (yitzhak12) [reply tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py:76]: Yes, I will change it.

**Comment by yitzhak12** [comment]
The pr validation passed: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/675/console.

**Review by keesturam — APPROVED** [review]
great PR. LGTM.

File: tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        def finalizer():
+            ocp_nodes = get_node_objs()
+            for n in ocp_nodes:
+                recover_node_to_ready_state(n)
+
+            config.switch_ctx(self.orig_index)
+
+        request.addfinalizer(finalizer)
+
+    def test_rolling_shutdown_and_recovery_in_controlled_fashion(self, nodes):
```
Comment (jilju) [inline tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py:77]: Please add polarion test case id.

Comment (yitzhak12) [reply tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery_ms.py:77]: Okay, that will do

**Review by jilju — APPROVED** [review]

**Review by keesturam — APPROVED** [review]

**Pull request merged**